src/components/Icon/Icon.tsx

```diff
@@ -207,25 +207,39 @@ function Icon({
     backdrop && styles.hasBackdrop,
   );
 
+  const defaultIconProps = {
+    className: styles.Svg,
+    viewBox: '0 0 20 20',
```

Comment:
svgr icons coming from polaris-icons already have a viewbox set. I think an svg without a viewbox won't really work, so should we be overwriting it here?

Reply:
We are currently telling people using SVGR to set a 20x20 viewbox, this means we don't have to anymore. I also tested this and it wasn't overwriting anything, if there's already a viewbox defined then it preserves that